forms/StyledSection/StyledSection.js

```diff
@@ -1,32 +1,92 @@
 import React from "react";
 import PropTypes from "prop-types";
+import styled from "styled-components";
 
 import Section from "../Section";
+import Heading from "../../composites/basic/Heading";
+import colors from "../../style-guide/colors.json";
+import { rgba } from "../../style-guide/helpers";
+import { Icon } from "../../composites/Plugin/Shared/components/Icon";
+import * as icons from "../../style-guide/svg";
+
+const StyledHeading = styled( Heading )``;
```

Comment:
Use Heading if no additional styles are needed.

Reply:
As far as I know, needs to be wrapped in a styled components to be able to do `& ${ StyledHeading } { ...` later.